pages/linux/grim.md

```diff
@@ -0,0 +1,28 @@
+# grim
+
+> Grab images (Screenshots) from a Wayland compositor.
+> More information: <https://github.com/emersion/grim>.
+
+- Screenshoot all outputs:
+
+`grim`
+
+- Screenshoot a specific output:
+
+`grim -o {{display_output}}`
```

Comment:
```suggestion
- Screenshoot a specific output:

`grim -o {{path/to/output_file}}`
```

If `-o` accepts output file you can change placeholder to what I suggest to make it compatible with [CLIP](https://github.com/command-line-interface-pages/syntax/blob/main/type-specific/cli.md) project placeholders.